README.md

```diff
@@ -39,67 +43,146 @@ Skopeo works with API V2 registries such as Docker registries, the Atomic regist
+        "27-armhfp",
+        "27-ppc64le",
+        "27-x86_64",
+        "27",
+        "28-aarch64",
+        "28-armhfp",
+        "28-ppc64le",
+        "28-x86_64",
+        "28",
+        "29-aarch64",
+        "29-ppc64le",
+        "29-s390x",
+        "29-x86_64",
+        "29",
+        "30-aarch64",
+        "30-ppc64le",
+        "30-s390x",
+        "30-x86_64",
+        "30",
         "latest",
-        "rawhide"
+        "rawhide",
+        "30-armhfp",
+        "29-armhfp",
+        "31-aarch64",
+        "31-x86_64",
+        "31",
+        "31-armhfp",
+        "31-s390x",
+        "31-ppc64le",
+        "32-aarch64",
+        "32-ppc64le",
+        "32-s390x",
+        "32-x86_64",
+        "32",
+        "33-aarch64",
+        "33-ppc64le",
+        "33-s390x",
+        "33-x86_64",
+        "33"
     ],
-    "Created": "2016-03-04T18:40:02.92155334Z",
-    "DockerVersion": "1.9.1",
-    "Labels": {},
+    "Created": "2020-04-29T06:48:16Z",
+    "DockerVersion": "1.10.1",
+    "Labels": {
+        "license": "MIT",
+        "name": "fedora",
+        "vendor": "Fedora Project",
+        "version": "32"
+    },
     "Architecture": "amd64",
     "Os": "linux",
     "Layers": [
-        "sha256:236608c7b546e2f4e7223526c74fc71470ba06d46ec82aeb402e704bfdee02a2",
-        "sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4"
+        "sha256:3088721d7dbf674fc0be64cd3cf00c25aab921cacf35fa0e7b1578500a3e1653"
+    ],
+    "Env": [
+        "DISTTAG=f32container",
+        "FGC=f32",
+        "container=oci"
+    ]
+}
+
+# Show configuration data about fedora lates
+$ skopeo inspect --config docker://registry.fedoraproject.org/fedora:latest  | jq
+{
+  "created": "2020-04-29T06:48:16Z",
+  "architecture": "amd64",
+  "os": "linux",
+  "config": {
+    "Env": [
+      "DISTTAG=f32container",
+      "FGC=f32",
+      "container=oci"
+    ],
+    "Cmd": [
+      "/bin/bash"
+    ],
+    "Labels": {
+      "license": "MIT",
+      "name": "fedora",
+      "vendor": "Fedora Project",
+      "version": "32"
+    }
+  },
+  "rootfs": {
+    "type": "layers",
+    "diff_ids": [
+      "sha256:a4c0fa2b217d3fd63d51e55a6fd59432e543d499c0df2b1acd48fbe424f2ddd1"
     ]
+  },
+  "history": [
+    {
+      "created": "2020-04-29T06:48:16Z",
+      "comment": "Created by Image Factory"
+    }
+  ]
 }
 
 # show unverifed image's digest
-$ skopeo inspect docker://docker.io/fedora:rawhide | jq '.Digest'
-"sha256:905b4846938c8aef94f52f3e41a11398ae5b40f5855fb0e40ed9c157e721d7f8"
+skopeo inspect docker://registry.fedoraproject.org/fedora:latest | jq '.Digest'
+"sha256:655721ff613ee766a4126cb5e0d5ae81598e1b0c3bcf7017c36c4d72cb092fe9"
 ```
 
 Copying images
 -
 `skopeo` can copy container images between various storage mechanisms, including:
-* Docker distribution based registries
+* Container registries
 
-  -  The Docker Hub, OpenShift, GCR, Artifactory, Quay ...
+  -  The Quay.io, Docker Hub, OpenShift, GCR, Artifactory ...
```

Comment:
Should we include the RHEL and Fedora registries too?

Reply:
No, I think the list is enough.  Not sure Openshift should be there either, but will leave.  Want to maintain this as not Red Hat specific.